### PR TITLE
[Core] Add two missing full stops to `licenseinfo`

### PR DESCRIPTION
**Pull request by Vexed01**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Add missing full stops in the `[p]licenseinfo` command.
Note that I'm not a lawyer so hopefully this doesn't cause any legal issues.

### CODE REVIEW

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -2658,10 +2658,10 @@ async def license_info_command(ctx):
     """
 
     message = (
-        "This bot is an instance of Red-DiscordBot (hereafter referred to as Red)\n"
+        "This bot is an instance of Red-DiscordBot (hereafter referred to as Red).\n"
```
Comment (Jackenmen) [inline redbot/core/core_commands.py]: I know it's not part of PR, but shouldn't `hereafter` be `hereinafter`?

Comment (Vexed01) [reply redbot/core/core_commands.py]: Yes, looking at that I think it should be.

**Review by Jackenmen — DISMISSED** [review]
Thanks!

**Review by Jackenmen — APPROVED** [review]
Thanks! (again 😄)

**Pull request merged**